README.md

```diff
@@ -59,6 +59,70 @@ f1.content = "The quick brown fox jumped over the lazy dog."
 bf1.save
 ```
 
+## Extending Directly Contained Derivatives
+
+Hydra Works comes with the ability to attach [Hydra Derivatives](https://github.com/projecthydra/hydra-derivatives) to a GenericFile. This allows a GenericFile of an image, video, audio file, or document type to directly contain appropriate derivative files such as thumbnail, extracted_text, mp3, ogg, mp4, and webm. The derivatives are configured in `lib/hydra/works/models/concerns/generic_file/contained_files.rb` and `lib/hydra/works/models/concerns/generic_file/derivatives.rb`.
+
+More directly contained derivatives can be configured for the Persist Derivatives service by extending `contained_files` and `makes_derivatives`, as long as the processor is defined in Hydra Derivatives.
+
+### contained_files
+```
+# lib/hydra/works/models/concerns/generic_file/contained_files.rb
```

Comment:
So I'm not sure either is necessary - works classes are meant to be inherited, no? So really I'd expect something like...

```ruby
class GenericFile
  include Hydra::Works::GenericWorkBehavior
  directly_contains_one :new_derivative_name, through: :files, type: ::RDF::URI("http://pcdm.org/use#ServiceFile"), class_name: "Hydra::PCDM::File"
  makes_derivatives do |obj|
    case obj.mime_type

      # create :new_derivative_name for pdf in additon to :thumbnail derivative
    when *pdf_mime_types
      obj.transform_file :original_file, { new_derivative_name: { format: 'png', size: '400x400' }, thumbnail: { format: 'jpg', size: '338x493' } }
    end
  end
end
```

I don't think this will work by the way - where's use#ServiceFile come from?